demo-simple/src/main/java/com/novoda/downloadmanager/demo/CustomNotificationCreator.java

```diff
@@ -4,25 +4,29 @@
 import android.content.Context;
 import android.support.v4.app.NotificationCompat;
 
-import com.novoda.notils.logger.simple.Log;
 import com.novoda.downloadmanager.DownloadBatchTitle;
+import com.novoda.downloadmanager.NotificationChannelCreator;
 import com.novoda.downloadmanager.NotificationCreator;
 import com.novoda.downloadmanager.NotificationInformation;
+import com.novoda.notils.logger.simple.Log;
 
 public class CustomNotificationCreator implements NotificationCreator {
+
     private static final int ID = 1;
     private static final boolean NOT_INDETERMINATE = false;
 
     private final Context context;
     private final int iconDrawable;
+    private final NotificationChannelCreator notificationChannelCreator;
 
-    public CustomNotificationCreator(Context context, int iconDrawable) {
+    public CustomNotificationCreator(Context context, int iconDrawable, NotificationChannelCreator notificationChannelCreator) {
         this.context = context;
         this.iconDrawable = iconDrawable;
+        this.notificationChannelCreator = notificationChannelCreator;
     }
 
     @Override
-    public NotificationInformation createNotification(DownloadBatchTitle downloadBatchTitle,
+    public NotificationInformation createNotification(String notificationChannelName, DownloadBatchTitle downloadBatchTitle,
```

Comment:
Sure we don't want to wrap this `String` in an object? 😅

Reply:
I can't tell if you are serious 😆

Reply:
second parameter in a new line ;)